docs/generate_checks.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+import yaml
+
+with open('checks.yaml') as f:
+    checks = yaml.safe_load(f)
+
+with open('checks.json', 'w') as f:
```

Comment:
is this even needed. ideal scenario would be scorecard tool, read from checks.yaml and add json as part of the latest.json ? if we use checks.json, it can be stale.

Reply:
+1. Since we are not using checks.json in this CL, I think we should just not generate it.